lib/std/c/haiku.zig

```diff
@@ -535,9 +569,6 @@ pub const O = struct {
     pub const RDWR = 0x0002;
     pub const ACCMODE = 0x0003;
 
-    pub const SHLOCK = 0x0010;
-    pub const EXLOCK = 0x0020;
-
     pub const CREAT = 0x0200;
     pub const EXCL = 0x0800;
     pub const NOCTTY = 0x8000;
```

Comment:
These constants, also, do not seem to properly correspond to ours. They should probably all be fixed. (I think a lot of the constants not touched in these commits have similar problems, maybe a general review of the constants in this file is in order.)

Reply:
see above

Reply:
updated